### PR TITLE
[TF ops] Use `_TFGetScalarOrDie` instead of `.scalarized()` in op implementations

### PR DESCRIPTION
**Pull request by rxwei**
Op implementation detail should use `_TFGetScalarOrDie` instead of `.scalarized()` to avoid unnecessary reshaping node generated.

### CODE REVIEW

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Review by dan-zheng — APPROVED** [review]

**Comment by dan-zheng** [comment]
Nice catch, thanks!

**Review by LaurentMazare — APPROVED** [review]

**Comment by rxwei** [comment]
@swift-ci please test tensorflow and merge

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Comment by rxwei** [comment]
@swift-ci please clean test tensorflow

File: stdlib/public/TensorFlow/Ops.swift [hunk 1]
```diff
@@ -904,13 +904,13 @@ public extension Tensor where Scalar : Numeric & Comparable {
   /// Returns the index of the maximum value of the flattened scalars.
   @_inlineable @inline(__always)
   func argmax() -> Int32 {
-    return self.flattened().argmax(squeezingAxis: 0).scalarized()
+    return _TFGetScalarOrDie(flattened().argmax(squeezingAxis: 0).handle)
```
Comment (mhong) [inline stdlib/public/TensorFlow/Ops.swift]: should we add some tests?

Comment (rxwei) [reply stdlib/public/TensorFlow/Ops.swift]: This patch is simply restoring the old graph generated by these ops. Tests will be added separately.

**Pull request merged**